integration/spark/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableDropHivePartitionCommand.scala

```diff
@@ -105,6 +109,28 @@ case class CarbonAlterTableDropHivePartitionCommand(
             ifExists,
             purge,
             retainData).run(sparkSession)
+
+          // move  the partition files to trash folder which are dropped
+          val droppedPartitionNames = partitions.map { partition =>
+            partition.spec.map { specs => specs._1 + "=" + specs._2}
+          }
+          val droppedPartitionName = droppedPartitionNames
```

Comment:
Taking Head Option to create Partition folder name in Trash is wrong here. For example, consider a case, where user performs drop multi-level partititon (i.e., `DROP PARTITION(deptname='Learning',doj='2015-12-01 00:00:00' )`) in this case, Trash folder also should have multi-level partition folder, to roll back drop partition if user wants. So, in TRash, please copy under respective multi-level partition folders

Reply:
Done and tested.